### PR TITLE
Contributor Guide: Document non-code contributions

### LINKED ISSUE #862: Contributor Experience: Create CONTRIBUTING.md
Contributing guidelines are provided in https://github.com/keptn/keptn.github.io/blob/master/README.md, but this location is not automatically displayed for newcomer contributors. It would help to improve the experience for newcomer documentation contributors.

### PR DESCRIPTION
**Pull request by oleg-nenashev**
This change initializes contributor guidelines for non-code contributions:

* Documentation
* Advocacy and Outreach
* Mentorship programs and internships
* Recommendations for monetary donations that we do not accept at the moment (right?)
* Placeholder for other types of contributions, e.g. design

Contributes to https://github.com/keptn/community/issues/82 . Also fixes #862 by moving the contributing content to a separate page.

Signed-off-by: Oleg Nenashev <o.v.nenashev@gmail.com>

## Screenshots

image

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Contribute to the Keptn Website
+
+This is the repository of the [Keptn](https://keptn.sh) website, which uses the [Hugo static website generator framework](http://gohugo.io).
+Keptn is a message-driven control-plane for application delivery and automated operations.
```
Comment (johannes-b) [inline CONTRIBUTING.md:4]: message-driven > event-driven

File: content/community/contributing/index.md [hunk 2]
```diff
@@ -23,5 +23,64 @@ If you want to contribute an **integration** to Keptn, please take a look at the
...
+We also have an account on [LFX Mentorship](https://mentorship.lfx.linuxfoundation.org/#projects_all)
+and can host our own mentorship programs for important topics contributing to the Keptn roadmap.
+
+If you are interested in mentorship programs as a potential mentor or mentee,
+reach out to the community in the `#advocacy-and-outreach` channel on the Keptn Slack.
+
+## Other contributions
+
+Any contributions will be appreciated even if they aren't listed here yet!
+We welcome contributors with any backgrounds and levels of experiences: code reviews, design and artwork, organizing events, content management, testing, issue triage, localization, etc.
```
Comment (johannes-b) [inline content/community/contributing/index.md:86]: experiences > experience

File: CONTRIBUTING.md (hunk 1 above)
Comment (oleg-nenashev) [reply CONTRIBUTING.md:4]:
```suggestion
Keptn is a event-driven control-plane for application delivery and automated operations.
```

File: content/community/contributing/index.md (hunk 2 above)
Comment (oleg-nenashev) [reply content/community/contributing/index.md:86]:
```suggestion
We welcome contributors with any backgrounds and levels of experience: code reviews, design and artwork, organizing events, content management, testing, issue triage, localization, etc.
```

**Review by johannes-b — APPROVED** [review]
LGTM

**Pull request merged**